tool/tsh/common/tsh.go

```diff
@@ -899,6 +900,7 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 	// play
 	play := app.Command("play", "Replay the recorded session (SSH, Kubernetes, App, DB).")
 	play.Flag("cluster", clusterHelp).Short('c').StringVar(&cf.SiteName)
+	play.Flag("speed", "Playback speed, applicable when streaming SSH or Kubernetes sessions.").Default("1x").EnumVar(&cf.PlaySpeed, "0.5x", "1x", "2x", "4x", "8x")
```

Comment:
Can we simply use float instead of string with a map converted to floats?

Reply:
We could, but I don't really want to allow a speed of 1.742236.

Reply:
`3.14159` is such a nice speed though 😉 I think there are sensible values which are disallowed now, and just using floats would allow them as well as make the code slightly simpler.

Not a big deal in any way, though.